packages/ui/src/components/Menu/SMenuItem.vue

```diff
@@ -16,14 +16,10 @@ const props = withDefaults(
 
 const emit = defineEmits<(event: 'click', value: HTMLElement) => void>()
 
-let isSelected = ref(false)
-let isMenuCollapsed = ref(false)
 const menuApi = useMenuApi()
 
-if (menuApi) {
-  isSelected = computed(() => menuApi.active.value === props.index)
-  isMenuCollapsed = computed(() => menuApi.collapsed.value)
-}
+const isSelected = computed(() => menuApi.active.value === props.index)
+const isMenuCollapsed = computed(() => menuApi.collapsed.value)
 
 function handleClick(event: HTMLElement) {
   if (menuApi) {
```

Comment:
About `event: HTMLElement`:

- AFAIK actual typing is `Event` or `ClickEvent`
- `event` is unused anyway. And I seems to me unclear why parent might need to get `Element` from the child on click.